start.sh

```diff
@@ -72,22 +76,18 @@ fi
 # This syntax substitutes False if null or unset
 if [ "${IS_CRON_POD:-"false",,}" == "false" ]; then
     if [ "${PTVSD_ENABLE:-"false",,}" == "false" ]; then
-        # Start Gunicorn processes
-        echo Starting Gunicorn for production
-
-        # application pod
+        echo "Starting Gunicorn for production"
         exec gunicorn dashboard.wsgi:application \
             --bind 0.0.0.0:${GUNICORN_PORT} \
             --workers="${GUNICORN_WORKERS}" \
             ${GUNICORN_RELOAD}
     else
-        # Currently ptvsd doesn't work with gunicorn
-        # https://github.com/Microsoft/vscode-python/issues/2138
-        echo Starting Runserver for development
-        export PYTHONPATH="/code:$PYTHONPATH"
-        export DJANGO_SETTINGS_MODULE=dashboard.settings
-        exec django-admin runserver --ptvsd 0.0.0.0:${GUNICORN_PORT}
-
+        echo "Starting Gunicorn for PTVSD debugging"
+        # Workers need to be set to 1 for PTVSD
+        exec gunicorn dashboard.wsgi:application \
+            --bind 0.0.0.0:${GUNICORN_PORT} \
+            --workers=1 \
+            ${GUNICORN_RELOAD}
```

Comment:
The logic here can be simplified for reading purpose

 ```
if [ "${PTVSD_ENABLE:-"false",,}" == "false" ]; then
       echo "Starting Gunicorn for production"
      
else
      echo "Starting Gunicorn for PTVSD debugging"
       GUNICORN_WORKERS = 1
    
fi
 exec gunicorn dashboard.wsgi:application \
            --bind 0.0.0.0:${GUNICORN_PORT} \
            --workers="${GUNICORN_WORKERS}" \
            ${GUNICORN_RELOAD}
```

Reply:
Yeah, that's a little easier to read, thanks!